packages/typescriptlang-org/src/copy/pt/documentation.ts

```diff
@@ -0,0 +1,80 @@
+    "Crie ferramentas de linha de comando que seus usuários amam",
+  doc_node_npm_gluegun_blurb:
+    "Uma maravilhosa coleção de ferramentas para construir aplicativos de linha de comando em Typescript",
+  doc_frameworks: "Frameworks Web",
+  doc_frameworks_angular_blurb:
+    "Torna escrever apps lindos em algo alegre e divertido",
+  doc_frameworks_ember_blurb: "Framework para desenvolvedores web ambiciosos",
+  doc_frameworks_react_blurb:
+    "Uma biblioteca JavaScript para construir interfaces de usuário",
+  doc_frameworks_vue_blurb: "O framework JavaScript progressivo",
+  doc_frameworks_ror_blurb: "Framework Web da convenção acima da configuração",
+  doc_frameworks_asp_blurb:
+    "Framework para aplicações modernas, conectadas à internet e baseadas na nuvem",
+  doc_apis: "APIs Node",
+  doc_apis_azure_blurb:
+    "Construa e distribua dentro do VS Code em questão de minutos",
+  doc_apis_feather_blurb: "Framework para aplicações em tempo real e APIs REST",
+  doc_apis_graphql_blurb: "Comece seu próprio servidor GraphQL em segundos",
+  doc_apis_nest_blurb:
+    "Uma framework Node.js progressiva para construir aplicações de servidor eficientes e escaláveis",
+  doc_apis_node_blurb:
+    "Um modelo bem documentado, cortesia do time do TypeScript",
+  doc_apis_wechat_blurb: "Use o JSSDK do WeChat com TypeScript",
+  doc_apis_loopback_blurb:
+    "Uma framework Node.js e Typescript altamente expansível para construir APIs e microserviços",
+  doc_apis_fastify_blurb:
+    "Uma framework web rápida e de baixo custo computacional para Node.js",
+  doc_react: "Projetos em React",
+  doc_react_create_blurb: "Crie uma aplicação web moderna com um só comando",
+  doc_react_gatsby_blurb:
+    "Ajuda desenvolvedores a construir sites e apps web incrivelmente rápidos",
+  doc_react_next_blurb: "A framework React",
+  doc_react_razzle_blurb:
+    "Aplicações universais em JavaScript, renderizadas em servidor - nenhuma configuração necessária",
+  doc_react_toolchains_title: "Ferramentas recomendadas",
+  doc_react_toolchains_blurb: "Recomendadas pelo time do React",
+  doc_apps: "Construíndo aplicativos",
+  doc_apps_electron_blurb:
+    "Construa aplicativos para desktop multi-plataforma com JavaScript, HTML e CSS",
+  doc_apps_expo_blurb: "A maneira mais rápida de se construir um app",
+  doc_apps_react_native_blurb: "Aprenda uma vez, escreva em qualquer lugar",
+  doc_apps_native_script_blurb:
+    "Framework de código aberto para construir aplicativos móveis verdadeiramente nativos",
+  doc_apps_make_code_blurb:
+    "Traga à vida ciência da computação para qualquer estudante com projetos divertidos",
+  doc_tooling: "Ferramentas",
+  doc_tooling_babel_blurb: "Use Javascript da próxima geração, hoje",
+  doc_tooling_parcel_blurb:
+    "Bundler de aplicações web incrivelmente rápido, sem configuração necessária",
+  doc_tooling_webpack_blurb: "Agrupe seus recursos, scripts, images e estilos",
+  doc_learn: "Já é familiar com Typescript?",
```

Comment:
```suggestion
  doc_learn: "Já conhece Typescript?",
```

Reply:
nice, this is actually hw I'd have said it